### DISCUSSION TITLE
Is it necessary to install Quarto separately from RStudio?

### DISCUSSION THREAD

**Discussion by jtr13** [quarto-dev/quarto-cli#5854] [Q&A] [answered]
### Description

It seems that the latest version of RStudio ships with Quarto. Do you recommend installing Quarto separately? Thanks.

**Comment by cscheid** [comment, marked as answer]
RStudio has shipped a version of Quarto for over a year now, so you definitely don't _need_ to install a separate version of Quarto.

With that said, if you're here and contributing with issues and discussions (as you demonstrably are :)), it's likely that you're a sufficiently advanced user of Quarto. You'll probably want to track Quarto development more closely than the RStudio release schedule, and then installing Quarto separately is definitely the way to go.

**Reply by jtr13** [reply]
Thanks, very helpful to know, that was not at all obvious to me. On the Get Started page the first step is to download Quarto so that is what I did. While showing students how to create Quarto websites I discovered to my surprise that they were able to render without installing Quarto. I'm asking so I can advise them well--it sounds like they would be fine using the version that comes with RStudio unless there's some significant gap in feature availability (which seems unlikely).

**Reply by cscheid** [reply]
> On the Get Started page the first step is to download Quarto so that is what I did

The thing to keep in mind is that the Quarto page is about Quarto, and it is very explicitly mindful of not assuming incoming users are R or RStudio users. The RStudio download page itself makes it fairly clear that Quarto is bundled. From [the download page](https://posit.co/download/rstudio-desktop/), click on Release notes, and note [the first lines](https://docs.posit.co/ide/news/#rstudio-ide) say

"New
RStudio IDE
Upgrade Quarto to 1.2.475 (#12887)"

**Reply by jtr13** [reply]
Thanks so much, really appreciate the explanation.
